searx/engines/public_domain_image_archive.py

```diff
@@ -40,7 +40,7 @@
 }
 
 base_url = 'https://oqi2j6v4iz-dsn.algolia.net'
-pdia_config_url = 'https://pdimagearchive.org/_astro/config.BiNvrvzG.js'
+pdia_config_url = 'https://pdimagearchive.org/_astro/InfiniteSearch.HlurxJ6Q.js'
```

Comment:
We should scrape this URL from https://pdimagearchive.org instead of hardcoding it.

Reply:
Completely correct, done!